main.go

```diff
@@ -26,11 +30,28 @@ func main() {
 		log.Fatal(err)
 	}
 
+	sigs := make(chan os.Signal, 1)
+	abort := make(chan bool, 1)
+	signal.Notify(sigs, syscall.SIGINT, syscall.SIGTERM)
+	go func() {
+		sig := <-sigs
+		io.WriteString(os.Stderr, fmt.Sprintf("Aborting! Caught signal \"%s\"\n", sig))
```

Comment:
i suspect you may want the simpler

`fmt.Fprintf(os.Stderr, "Aborting! Caught Signal \"%s\"\n", sig))`

(ditto down below, `fmt.Fprint(os.Stderr, "Cleaning up...\n")`